src/13_file_io.py

```diff
@@ -8,11 +8,21 @@
 # Open up the "foo.txt" file (which already exists) for reading
 # Print all the contents of the file, then close the file
 
+f = open('foo.txt')
+read_data = f.read()
+print(read_data)
+f.close()
+
```

Comment:
This way is good also, here is just another way to do it.

with open("foo.txt") as fp:
  for line in fp:
    print(line)

fp.close()